src/gui/sysConf.cpp

```diff
@@ -2231,6 +2231,7 @@ bool FurnaceGUI::drawSysConf(int chan, int sysPos, DivSystem type, DivConfig& fl
     case DIV_SYSTEM_C219:
       break;
     case DIV_SYSTEM_YMU759:
+    case DIV_SYSTEM_ESFM:
```

Comment:
Are you sure?

Reply:
For now, there's doesn't seem to be a need for any configuration parameters, as there are no known differences between ESFM implementations in different ESS chipsets. They all run at the same clock speed and same output rate, and seem to have the same features.

One thing we're investigating, however, is a hardware bug that causes the output to overflow. I'll likely add an option in ESFMu to enable emulation of it once it's understood and properly emulated, and that could be a contender for a config flag. It might even be a thing that happens in some revisions and not in others, and might be a thing that depends on factors that are external to the ESFM synth itself (the mixing section on the ESS chipset). The thing is, adding this config flag right now might cause confusion since it'll currently do nothing as it's not yet emulated.

Reply:
I see.
Note that Furnace allows some chips to run at arbitrary clock rates, even if only one rate is known to be used.